### PR TITLE
feat: add `--manifest` flag to `svc show`

### PR DESCRIPTION
**Pull request by efekarakus**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/cli/env_show.go [hunk 1]
```diff
@@ -183,8 +177,10 @@ func buildEnvShowCmd() *cobra.Command {
 		Long:  "Shows info about a deployed environment, including region, account ID, and services.",
 
 		Example: `
-  Shows info about the environment "test".
-  /code $ copilot env show -n test`,
+  Print configuration for the "test" environment "test".
```
Comment (huanjani) [inline internal/pkg/cli/env_show.go]: nit:
```suggestion
  Print configuration for the environment "test".
```
or
```suggestion
  Print configuration for the "test" environment.
```

File: internal/pkg/cli/env_show.go [hunk 2]
```diff
@@ -198,6 +194,8 @@ func buildEnvShowCmd() *cobra.Command {
 	cmd.Flags().BoolVar(&vars.shouldOutputJSON, jsonFlag, false, jsonFlagDescription)
 	cmd.Flags().BoolVar(&vars.shouldOutputResources, resourcesFlag, false, envResourcesFlagDescription)
 	cmd.Flags().BoolVar(&vars.shouldOutputManifest, manifestFlag, false, manifestFlagDescription)
-	_ = cmd.Flags().MarkHidden(manifestFlag)
+
+	cmd.MarkFlagsMutuallyExclusive(jsonFlag, manifestFlag)
```
Comment (huanjani) [inline internal/pkg/cli/env_show.go:198]: pretty cool!

File: internal/pkg/cli/svc_show.go [hunk 3]
```diff
@@ -198,6 +206,20 @@ func (o *showSvcOpts) getTargetSvc() (*config.Workload, error) {
 	return o.targetSvc, nil
 }
 
+func (o *showSvcOpts) writeManifest() error {
+	out, err := o.describer.Manifest(o.outputManifestForEnv)
+	if err != nil {
+		var errNotFound *describe.ErrManifestNotFoundInTemplate
+		if errors.As(err, &errNotFound) {
+			log.Infof("You must deploy a new version of your service with %s before printing its manifest.\n",
+				color.HighlightCode("copilot deploy"))
```
Comment (huanjani) [inline internal/pkg/cli/svc_show.go]: I wonder if we could do:
```suggestion
				color.HighlightCode(fmt.Sprintf("copilot svc deploy -n %s", o.svcName))
```
?

File: internal/pkg/describe/service.go [hunk 4]
```diff
@@ -124,7 +125,7 @@ func (d *serviceStackDescriber) Params() (map[string]string, error) {
 	return descr.Parameters, nil
 }
 
-// Params returns the outputs of the service stack.
+// Outputs returns the outputs of the service stack.
```
Comment (huanjani) [inline internal/pkg/describe/service.go:128]: 😂

**Review by huanjani — APPROVED** [review]
Just a couple nits; otherwise, looks good!

File: internal/pkg/cli/svc_show.go (hunk 3 above)
Comment (dannyrandall) [reply internal/pkg/cli/svc_show.go]: we could also add the env!

**Review by dannyrandall — APPROVED** [review]

File: internal/pkg/cli/svc_show.go (hunk 3 above)
Comment (efekarakus) [reply internal/pkg/cli/svc_show.go]: Done!

**Pull request merged**